builder/java/rules.bzl

```diff
@@ -111,3 +130,62 @@ def native_dep_for_host_platform(name):
          "@vaticle_bazel_distribution//platform:is_windows": [name + "-windows"],
          "//conditions:default": ["INVALID"],
      })
+
+def typedb_java_test(name, server_artifacts, console_artifacts = {},
+                      native_libraries_deps = [], deps = [], classpath_resources = [], data = [], args = [], **kwargs):
+    native_server_artifact_paths, native_server_artifact_labels = native_artifact_paths_and_labels(server_artifacts)
+    native_console_artifact_paths, native_console_artifact_labels = native_artifact_paths_and_labels(console_artifacts, mandatory = False)
+    native_dependencies = get_native_dependencies(native_libraries_deps)
+
+    native.java_test(
+        name = name,
+        deps = deps + native_dependencies,
```

Comment:
Note: we no longer implicitly depend on the runner library, as there is no longer a single runner library.